cron/controller/Dockerfile

```diff
@@ -27,5 +27,6 @@ RUN CGO_ENABLED=0 make build-pubsub
 
 FROM gcr.io/distroless/base:nonroot@sha256:bc84925113289d139a9ef2f309f0dd7ac46ea7b786f172ba9084ffdb4cbd9490
 COPY ./cron/data/projects.csv cron/data/projects.csv
+COPY ./cron/data/projects.release.csv cron/data/projects.release.csv
```

Comment:
nit: just `COPY ./cron/data/projects*csv cron/data/` to make it simpler and to avoid the extra layer.

Reply:
Done.